.github/workflows/miri.yml

```diff
@@ -0,0 +1,21 @@
+name: Miri test
```

Comment:
Since right now we're scoped to `profile::internal::observation`, I would suggest maybe including that here on the name (to avoid a misleading sense of safety -- oh we're running miri on stuff)?

Reply:
Done